src/style/colors.ts

```diff
@@ -11,7 +12,7 @@ export class Colors {
   [key: string]: any;
 
   constructor() {
-    const colors = Object.assign(colorsPalette, themeColors);
+    const colors = Object.assign(designTokens, themeColors);
```

Comment:
See my previous comment on separating between the tokens and colorsPalette, here you can join them together instead of include the colorsPalette in the tokens